ifrs17-template/Test/IfrsVariablesTest.ipynb

```diff
@@ -192,11 +247,40 @@
                 "\n        foreach (var variable in computedNotExpected)",
                 "\n            errors.Add(new BenchmarkTestResult(\"Extra computed variable for: Partition \" + variable.Partition + \", \" + variable.ToIdentityString()));",
                 "\n",
-                "\n    var comparer = EqualityComparer<IfrsVariable>.Instance; ",
+                "\n    var comparer = IfrsVariableComparer.Instance(true); ",
```

Comment:
why this change?

Reply:
@dcolleoni  I didn't understand how the EqualityComparer functions and found at the Utils another class IfrsVariableComparer. My main motivation to change the comparer was in necessity to compare projected values, though I realized now that the EqualityComparer checks the equality of all properties. I check it now and change it back.

Reply:
@dcolleoni Done.